docs/release_notes.rst

```diff
@@ -11,7 +11,7 @@ Notes for major and minor releases. Notes for patch releases are referred to the
    (date of release, format YYYY-MM-DD)
 
    **Of interest to the User:**
-
+   - PR #69 Removes `Show 2D Plots` option and instead always generates them. Displays a message on errors.
```

Comment:
update your feature branch with latest `next` and move this entry to the notes for release `2.8.0`